pages/docs/tutorials/native/first-kotlin-native-app.md

```diff
@@ -0,0 +1,88 @@
+---
+type: tutorial
+layout: tutorial
+title:  "First Kotlin/Native Application"
+description: "A look at how to compile our first Kotlin/Native application"
+authors: Hadi Hariri 
+date: 2017-12-03
+showAuthorInfo: false
+---
+
+
+In this tutorial we'll see how to
+
+* [Obtain the Kotlin/Native compiler](#Obtaining-the-compiler)
+* [Write the application](#Creating-Hello-Kotlin)
+* [Compile and examine output](#Compiling-and-examining-output)
+* [Run the application](#Running-the-application)
+
+
+## Obtaining the compiler
+
+Kotlin/Native is available for different macOS, Linux and Windows. Depending on the operating system we're working on, we'll need to download
+the correct compiler. While cross-platform compilation is possible (i.e. using one platform to compile for another), in this first tutorial
+we're going to compile for the same operating system we're running on. In our case this will be macOS. 
+
+We can obtain the latest version of the compiler from the [GitHub releases page](https://github.com/JetBrains/kotlin-native/releases).
+
+Once downloaded, we can uncompress it in any folder. For convenience it's useful to add the `bin` folder to the system path so that we can invoke the 
+compiler from any location.
```

Comment:
Please note, that Kotlin/Native compiler depends on the JVM. Produced binaries do not, but compiler itself needs Java 8 installed.